src/basic/unit-file.c

```diff
@@ -361,7 +363,7 @@ int unit_file_resolve_symlink(
         }
 
         *ret_destination = TAKE_PTR(dst);
-        return 0;
+        return tail ? false : true;  /* true if linked unit file */
```

Comment:
!tail?

Reply:
Unless I'm utterly confused, this is correct. If we have `tail`, then it's an alias, and this function is supposed to return false.

Reply:
yes i am just saying, why write such a complex ternary op, if `!tail` also works, and it is much shorter.

Reply:
Oh, I see. I changed it like this now. I thought that the explicit form would be clearer, but guess not.